modules/blocks/verify.js

```diff
@@ -787,6 +790,9 @@ Verify.prototype.processBlock = function(block, broadcast, saveBlock, cb) {
 			verifyBlock(seriesCb) {
 				__private.verifyBlock(block, seriesCb);
 			},
+			broadcastBlock(seriesCb) {
+				__private.broadcastBlock(block, broadcast, seriesCb);
+			},
```

Comment:
This broadcast can be done parallel to `processBlock` to increase the performance.